src/components/TaskAnalysisTable/TaskAnalysisTable.jsx

```diff
@@ -378,7 +386,7 @@ const setupColumnTypes = (props, taskBaseRoute, manager, data, openComments) =>
       const taskLink = (
         <div className="row-controls-column mr-links-green-lighter">
           <Link
-            to={`/challenge/${t.parentId}/task/${t.id}`}
+            to={`/challenge/${t.parentId || t.parent}/task/${t.id}`}
```

Comment:
Nit: I think you want `t.parentId ?? t.parent` here, in order to avoid a bug if `t.parentId` is 0.